docs/routes.js

```diff
@@ -32,6 +33,12 @@ export const routes = [
         path: '/Form',
         component: Form
     },
+    {
+        key: 'upload',
+        label: 'Upload',
+        path: '/Upload',
+        component: Upload
+    },
```

Comment:
Ich finde die Upload Component braucht keinen eigenen Navigationspunkt und könnte unter Form stehen

Reply:
done

Reply:
Das Example File sollte dann aber auch in `Form` verschoben werden

Reply:
done